tests/test_unclassified_job.py

```diff
@@ -21,6 +21,8 @@ def test_unclassified_failure(self, mozwebqa):
         # Open resultset page and search for next unclassified failure
         resultset_page = ResultsetPage(mozwebqa)
         resultset_page.go_to_page()
+        Assert.greater_equal("resultset_page.return_unclassified_failure_count", "1")
```

Comment:
This isn't doing what you're expecting. For comparing if something is greater or equal your values should be numbers. I would recommend `return_unclassified_failure_count` return an integer, and then you also need to change the value you're comparing it to an integer.